bundles/org.openhab.binding.mercedesme/src/main/java/org/openhab/binding/mercedesme/internal/AccountHandler.java

```diff
@@ -0,0 +1,209 @@
+/**
```

Comment:
This class could be moved into a **handler** subdirectory (often structured like this in other bindings).

Reply:
Moved